heisskleber/console/sink.py

```diff
@@ -0,0 +1,39 @@
+import json
+import sys
+import time
+from typing import TextIO
+
+from heisskleber.core.types import AsyncSink, Serializable, Sink
+
+
+def pretty_print(data: dict[str, Serializable]) -> str:
+    return json.dumps(data, indent=4)
+
+
+class ConsoleSink(Sink):
+    def __init__(self, stream: TextIO = sys.stdout, pretty: bool = False):
+        self.stream = stream
+        self.print = pretty_print if pretty else json.dumps
+
+    def send(self, data: dict[str, Serializable], topic: str) -> None:
+        self.stream.write(self.print(data))
+        self.stream.write("\n")
+        self.stream.flush()
```

Comment:
I would avoid calling `.flush()` as this can lead to massively increased system load when forced

Reply:
Will change, thanks!